solr/bin/solr.in.sh

```diff
@@ -82,7 +82,7 @@
 # Set to true to activate the JMX RMI connector to allow remote JMX client applications
 # to monitor the JVM hosting Solr; set to "false" to disable that behavior
 # (false is recommended in production environments)
-ENABLE_REMOTE_JMX_OPTS="true"
+# ENABLE_REMOTE_JMX_OPTS="true"
```

Comment:
Actually the commented text should say "false" as it was in 8.0. Commented defaults should reflect the implicit defaults. Can you fix? Please also add a CHANGES.TXT entry about the fix.

Reply:
I made those changes. review at your leisure

Reply:
Thanks. I meant to *leave* the line commented, but change from true to false..

Reply:
keeping you on your toes - /facepalm